packages/graphiql-plugin-utils/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@graphiql/plugin-utils",
+  "description": "shared tooling for building graphiql plugins",
+  "version": "0.0.1",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "cjs/vite-plugin.js",
+  "module": "dist/vite-plugin.js",
+  "types": "types/vite-plugin.d.ts",
+  "exports": {
+    "./base.tsconfig.json": "./base.tsconfig.json",
+    "./vite.config.tsconfig.json": "./vite.config.tsconfig.json",
+    ".": {
+      "types": "./types/vite-plugin.d.ts",
+      "require": "./cjs/vite-plugin.js",
+      "import": "./dist/vite-plugin.js"
+    }
+  },
+  "files": [
+    "*.json",
+    "dist",
+    "cjs"
+  ],
+  "scripts": {
+    "build": "tsc && tsc --module CommonJS --outDir cjs"
```

Comment:
pls do not use tsc, as we discussed already, use tsup

Reply:
we will introduce tsup in another PR

Reply:
why not just make it here? you don't need even tsup config, as you said

Reply:
because the plan is to use tsup to replace vite in most places. we will continue to use tsc for project references, because tsc --watch or tsup --watch are too slow across the whole monorepo

Reply:
You don't need tsc refererences for this package, and even `tsc/tsup --watch`, you can already experiment with adding `tsup` here